src/test/java/com/artipie/rpm/meta/XmlOthersTest.java

```diff
@@ -0,0 +1,122 @@
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+/**
+ * Testcase for {@link XmlOthers} class.
+ * @since 0.1
+ */
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class XmlOthersTest {
+
+    /**
+     * Path of the file.
+     */
+    private static String path = "src/test/resources/other.xml";
+
+    /**
+     * The XmlOthers object.
+     */
+    private static XmlOthers xmlOthers = new XmlOthers(Path.of(XmlOthersTest.path));
+
+    /**
+     * File object for the created file.
+     */
+    private static File file = new File(XmlOthersTest.path);
+
+    /**
+     * Expected content of the file.
+     * @checkstyle StringLiteralsConcatenationCheck (20 lines)
+     */
+    private static String expectedOut =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n"
+        + "<otherdata xmlns=\"http://linux.duke.edu/metadata/other\" packages=\"1\">\n"
+        + "    <package name=\"aom\" "
+        + "pkgid=\"7eaefd1cb4f9740558da7f12f9cb5a6141a47f5d064a98d46c29959869af1a44\""
+        + " arch=\"aarch64\">\n"
+        + "        <version ver=\"1.0.0\" rel=\"8.20190810git9666276.el8\" epoch=\"0\"/>\n"
+        + "        <changelog>NOT_IMPLEMENTED</changelog>\n"
+        + "    </package>\n"
+        + "</otherdata>\n";
+
+    /**
+     * Setting up the file and insert some test packages.
+     * @throws Exception
+     * @todo #80:30min after implementing the changelog, refactor this to add actual changelog
+     *  this should also include changing the {@code expectedOut} to match the added changelog
+     */
+    @BeforeAll
+    public void setUp() throws Exception {
+        XmlOthersTest.xmlOthers.startPackages();
+        final XmlOthers.Package pack = XmlOthersTest.xmlOthers.addPackage(
+            "aom",
+            "aarch64",
+            "7eaefd1cb4f9740558da7f12f9cb5a6141a47f5d064a98d46c29959869af1a44"
+        );
+        pack.version(0, "1.0.0", "8.20190810git9666276.el8");
+        pack.changelog();
+        pack.close();
+        XmlOthersTest.xmlOthers.close();
+    }
+
+    /**
+     * Delete the created file after test is done.
+     * @throws Exception
+     */
+    @AfterAll
+    public void tearDown() throws Exception {
+        XmlOthersTest.file.delete();
+    }
+
+    /**
+     * Test the created file content is the same as expected.
+     */
+    @Test
+    public void addingAPackage() {
+        try {
+            final String content = Files.readString(Path.of(XmlOthersTest.path));
+            MatcherAssert.assertThat(
+                "file created",
+                content,
+                IsEqual.equalTo(XmlOthersTest.expectedOut)
+            );
+        } catch (final IOException exception) {
+            MatcherAssert.assertThat(
```

Comment:
@ammaratef45 there are not need to assert exception here, it will fail unit test automatically if thrown. Just add `throws Exception` declaration to method signature and it will work